panther_lights/README.md

```diff
@@ -189,8 +194,8 @@ Modify also `command` to use user animations:
 command: roslaunch panther_bringup bringup.launch user_animations_file:=/user_animations.yaml --wait
 ```
 
-:warning: **Warning**
-While using docker you will only be able to find packages that are within that docker. Only images from packages that were built inside that docker imge can be  found using `$(find my_package)` syntax. Global paths work normally, but will refere to paths inside docker container.
+:warning: **Warning**:
+While using docker you will only be able to find packages that are within that docker. Only images from packages that were built inside that docker image can be found using `$(find my_package)` syntax. Global paths work normally, but will refere to paths inside docker container.
```

Comment:
```suggestion
> **Warning**
>
> While using docker you will only be able to find packages that are within that docker. Only images from packages that were built inside that docker image can be found using `$(find my_package)` syntax. Global paths work normally, but will refere to paths inside docker container.
```

This also applies to all Warnings and Notes. For consistency with other projects in our company, I recommend something like this:
```markdown
> **Note**
>
> ...
```
or
```markdown
> **Warning**
>
> ...
```